### PR TITLE
About plugin using virtual DOM APIs

### PR DESCRIPTION
**Pull request by charnpreetsingh**
About plugin now uses virtual DOM APIs to match design patterns used in the rest of JupyterLab.

* CSS classes not being used were deleted.
* CSS classes renamed to follow style

### CODE REVIEW

File: src/about/index.ts [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+ * The class name added to the filebrowser image.
+ */
+const FILEBROWSER_IMAGE_CLASS = 'jp-About-fb';
+
+/**
+ * The class name added to the notebook image.
+ */
+const NOTEBOOK_IMAGE_CLASS = 'jp-About-nb';
+
+export
```
Comment (blink1073) [inline src/about/index.ts]: Missing a docstring

File: src/about/index.ts [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+class AboutModel extends VDomModel {
+  constructor() {
+    super();
+    this._title = 'Welcome to the JupyterLab Alpha preview';
+    this._headerText = [
```
Comment (blink1073) [inline src/about/index.ts]: We can avoid having the private variables by setting the public values here and having `readonly title: string;` etc. on the class.

File: src/about/index.ts [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+  private _title: string;
+  private _headerText: string[];
+  private _pluginHeaders: string[];
+  private _mainAreaDesc: string[];
+  private _filebrowserDesc: string[];
+  private _commandPaletteDesc: string[];
+  private _notebookDesc: string[];
+}
+
+export
```
Comment (blink1073) [inline src/about/index.ts]: Missing docstring for the class and its methods.

File: src/about/plugin.ts [hunk 4]
```diff
@@ -2,21 +2,17 @@
...
-
-import {
   JupyterLab, JupyterLabPlugin
 } from '../application';
 
 import {
   ICommandPalette
 } from '../commandpalette';
 
+// add in other imports here
```
Comment (blink1073) [inline src/about/plugin.ts]: Is this comment intentional?

**Comment by charnpreetsingh** [comment]
@blink1073 changes made!

**Comment by blink1073** [comment]
Great, thanks!

**Pull request merged**